gatsby-config.ts

```diff
@@ -30,7 +30,6 @@ module.exports = {
     siteUrl: hostedUrl,
   },
   plugins: [
-    "gatsby-plugin-react-helmet",
```

Comment:
we are not using this internally?

Reply:
No, we are using react-helmet and "gatsby-plugin-react-helmet",
 was just added in plugins but not in use. @surajcontentstack